avae/settings.py

```diff
@@ -17,6 +17,7 @@
 VIS_SIM = False
 VIS_DYN = False
 VIS_POSE_CLASS = None
+VIS_Z_N_INT = "1,10"
```

Comment:
same comment as above, this should be general to all models so if not defined should be an empty list

Reply:
i miss understood this, should be ok